### PR TITLE
Exclude cases from showing on tupaia frontend

### PR DESCRIPTION
**Pull request by edmofro**
Relates to https://github.com/beyondessential/tupaia/pull/1577#issuecomment-724264862

### CODE REVIEW

File: packages/web-config-server/src/apiV1/organisationUnit.js [hunk 1]
```diff
@@ -63,11 +68,13 @@ export default class extends RouteHandler {
   }
 
   async filterForAccess(entities) {
-    return (await Promise.all(
-      entities
-        .filter(entity => entity)
-        .map(async entity => (await this.checkUserHasEntityAccess(entity)) && entity),
-    )).filter(entity => entity);
+    return (
```
Comment (edmofro) [inline packages/web-config-server/src/apiV1/organisationUnit.js:71]: Prettier made this change

File: packages/web-config-server/src/apiV1/organisationUnitSearch.js [hunk 2]
```diff
@@ -42,7 +42,12 @@ export default class extends RouteHandler {
     const project = await this.fetchProject();
     const projectEntity = await project.entity();
 
-    const allEntities = await projectEntity.getDescendants(project.entity_hierarchy_id);
+    const allEntities = await projectEntity.getDescendants(project.entity_hierarchy_id, {
+      type: {
+        comparator: 'not in',
+        comparisonValue: this.models.entity.typesExcludedFromWebFrontend,
+      },
+    });
```
Comment (kael89) [inline packages/web-config-server/src/apiV1/organisationUnitSearch.js:50]: This query is the same with the other one in `organisationUnit.js`, does it make sense to extract them in a common method (eg `getDescendantsForFrontEnd`)?

From one hand, we shouldn't go very far re-defining methods for every param combination, on the other this encapsulates specific business logic so it's probably worth it. I will leave the decision to you! 😎

**Review by kael89 — APPROVED** [review]
Thanks for taking care of this @edmofro !

File: packages/web-config-server/src/apiV1/organisationUnitSearch.js (hunk 2 above)
Comment (edmofro) [reply packages/web-config-server/src/apiV1/organisationUnitSearch.js:50]: Yeah it's a question I thought about, but only for 30 seconds, and then decided it's simple enough to repeat both places (and actually reads fairly fluently to my eyes)

**Pull request merged**